Bio/SeqFeature.py

```diff
@@ -1640,15 +1669,17 @@ def position(self):
 
     @property
     def extension(self):
-        """Legacy attribute to get extension (from left to right) as an integer (OBSOLETE)."""
+        """Legacy attribute to get the dimension (from left to right) as an integer (OBSOLETE)."""
         return self._right - self._left
 
     def _shift(self, offset):
+        """Legacy attribute to get extension (zero) as integer (PRIVATE)."""
```

Comment:
Copy and paste error? See the other ``_shift`` methods where you had a sensible docstring, e.g.

``Return a copy of the position object with its location shifted (PRIVATE).``